plugins/inventory/linode.py

```diff
@@ -282,26 +294,10 @@ def _get_query_options(self, config_data):
 
         return regions, types, tags
 
-    def verify_file(self, path):
-        """Verify the Linode configuration file."""
-        if super(InventoryModule, self).verify_file(path):
-            endings = ('linode.yaml', 'linode.yml')
-            if any((path.endswith(ending) for ending in endings)):
-                return True
-        return False
-
-    def parse(self, inventory, loader, path, cache=True):
-        """Dynamically parse Linode the cloud inventory."""
-        super(InventoryModule, self).parse(inventory, loader, path)
-
-        if not HAS_LINODE:
-            raise AnsibleError('the Linode dynamic inventory plugin requires linode_api4.')
-
-        config_data = self._read_config_data(path)
-        self._build_client(loader)
-
-        self._get_instances_inventory()
+    def _cacheable_inventory(self):
+        return [i._raw_json for i in self.instances]
```

Comment:
`_raw_json` seems like something internal that can break in any update of the library used. Is there a better way to get hold of the raw data?

Reply:
I actually talked this over with our DX team @ Linode, and `_raw_json` is leveraged frequently across our libraries. So I am confident that we will not remove it from the Linode APIv4 Python package.